openshift_tools/monitoring/zagg_sender.py

```diff
@@ -20,83 +20,47 @@
     zs.send_metrics()
 """
 
+import json
 from openshift_tools.monitoring.metricmanager import UniqueMetric
 from openshift_tools.monitoring.zagg_client import ZaggClient
 from openshift_tools.monitoring.zagg_common import ZaggConnection
-import json
-import os
-import yaml
+from openshift_tools.monitoring.generic_metric_sender import GenericMetricSender
 
-class ZaggSenderException(Exception):
-    '''
-        ZabbixSenderException
-        Exists to propagate errors up from the api
-    '''
-    pass
-
-class ZaggSender(object):
+class ZaggSender(GenericMetricSender):
     """
     collect and create UniqueMetrics and send them to Zagg
     """
 
-    def __init__(self, host=None, zagg_connection=None, verbose=False, debug=False):
+    def __init__(self, config_file=None, host=None, zagg_connection=None, verbose=False, debug=False):
```

Comment:
Adding 'config_file' to the front of the parameter list breaks the pre-existing ops-zagg-client. Is there a reason for adding the parameter to the front?